node/lib/task.ts

```diff
@@ -171,9 +200,10 @@ export function setResourcePath(path: string): void {
         resourceFile = path;
         debug('set resource file to: ' + resourceFile);
 
-        var locStrs = loadLocStrings(resourceFile);
+        resourceCulture = getVariable('system.culture') || resourceCulture;
```

Comment:
Will resourceCulture ever end up empty/null after this statement? If so, have you thought through the impact - i.e. will it error out later?